com.coala.core/src/com/coala/core/handlers/RuncoafileHandler.java

```diff
@@ -57,7 +58,8 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
       if (folder != null && project != null) {
         ExternalUtils.runcoafile(folder, project);
       } else {
-        System.out.println("Couldn't run analysis using .coafile");
+        MessageDialog.openError(null, "coafile not found",
+            "This coafile couldn't be found in this directory. Please create one.");
```

Comment:
IMO "this directory" is more appropriate since the user will be shown this error after they select a directory from the Project Explorer context menu.